core/services/synchronization/telem/telem.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
We should add a makefile target to build the resulting implementations for this. I have made an example here: https://github.com/smartcontractkit/offchain-reporting/pull/41/files#diff-e3445fc75aa9c3e4a60fbe5394dcce12693022018216a8fbe0000fe9952850a6

Reply:
Added a new target to the makefile